src/applications/vaos/tests/utils/eligibility.unit.spec.js

```diff
@@ -20,11 +20,8 @@ import { VHA_FHIR_ID } from '../../utils/constants';
 
 describe('VAOS scheduling eligibility logic', () => {
   describe('getEligibilityData', () => {
-    before(() => {
-      mockFetch();
-    });
-
     beforeEach(() => {
+      mockFetch();
```

Comment:
Due to a change I made to `resetFetch()` we need to make sure that `mockFetch()` is called before `setFetchJSONResponse()`. In this case the issue is that `resetFetch()` was called after each test but `mockFetch()` was only called once before the entire test suite ran.

Reply:
@narin @jbalboni I had to make this small change to your test file.